src/modules/Experiences/ExperiencesList/helpers.js

```diff
@@ -84,4 +84,61 @@ const configureForm = (expType) => {
   }
 };
 
-export { formatMonthYearDate, getWorkLength, configureForm };
+const monthScore = {
+  January: 1,
+  February: 2,
+  March: 3,
+  April: 4,
+  May: 5,
+  June: 6,
+  July: 7,
+  August: 8,
+  September: 9,
+  October: 10,
+  November: 11,
+  December: 12,
+};
+
+const getMonthScore = (experiences) => {
+  experiences.map((exp) => {
+    exp.start_month_score = monthScore[exp.start_month];
+
+    // exclude end_month === 'none'
+    if (exp.is_current === false) {
+      exp.end_month_score = monthScore[exp.end_month];
+    }
+  });
+
+  return experiences;
+};
+
+const sortExperiences = (experiences) => {
+  const experiencesWithScores = getMonthScore(experiences);
+
+  const sortedExperiences = experiencesWithScores.sort(function(exp1, exp2) {
+    // if is_current is True or have the same end month/year
+    // sort from start month/year (newest experience)
+    if (exp1.end_month === exp2.end_month && exp1.end_year === exp2.end_year) {
+      if (exp1.start_year > exp2.start_year) return -1;
```

Comment:
When you're comparing numeric values, you can just do `return exp2.start_year - exp1.start_year`. Of course, that assumes you don't want to fall back to another comparison if they are equal

Reply:
So much better this way. Thanks.